Documentation/ApiOverview/Localization/LocalizationApi/LanguageService.rst

```diff
@@ -12,5 +12,37 @@ be created directly, therefore its constructor is internal. Create a
 
 In the backend context a :php:`LanguageService` is stored in the global
 variable :php:`$GLOBALS['LANG']`.
+In the frontend a :php:`LanguageService` can be accessed via the contentObject:
+
+..  code-block:: php
+    :caption: Classes/Controller/ExampleController.php
+
+    use Psr\Http\Message\ServerRequestInterface;
+    use TYPO3\CMS\Core\Localization\LanguageServiceFactory;
+    use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+    class ExampleController {
+        proteced ServerRequestInterface $request;
```

Comment:
```suggestion
        private ServerRequestInterface $request;
```

We should differentiate between Extbase-based controllers and others: Extbase-based controllers already provide $this->request. This should be mentioned IMHO.

Reply:
Absolutely correct. As Franz does not make use of `extends ActionController` his example could be a target of a backend route or a call from COR->callUserFunc. So, yes, would be good to inform the reader that this example is a non-extbase example.

Reply:
In a backend route you have $GLOBALS['LANG'] - so no need for LanguageServiceFactory.

And: The reader might only read ...Controller and miss the "extends ActionController" -> a clarification protects for time-consuming error searching.